nemo/collections/nlp/modules/common/lm_utils.py

```diff
@@ -88,7 +88,7 @@ def get_lm_model(
             config_dict=config_dict, config_file=config_file, pretrained_model_name=pretrained_model_name,
         )
 
-    if checkpoint_file:
+    if checkpoint_file and os.path.exists(checkpoint_file):
```

Comment:
How about adding a warning when the path does not exists?

Reply:
The previous code would have failed but the current one would be silence.

Reply:
Actually I think we should raise an error as checkpoint_file is specified while it does not exists?

Reply:
can't do that - it will make some pre-trained models unusable. Specifically, the ones from NGC :)